src/DataCollectors/TraceDataCollector/VanguardCollector/DefaultCodeCoverageConfig.xml

```diff
@@ -23,7 +23,7 @@
         <ModulePath>.*clrjit.dll$</ModulePath>
         <ModulePath>.*clrjit.ni.dll$</ModulePath>
         <ModulePath>.*mscoree.dll$</ModulePath>
-        <ModulePath>.*mscoreei.dll$</ModulePath>
+        <!--<ModulePath>.*mscoreei.dll$</ModulePath> -->
```

Comment:
Please add a comment & link to task

Reply:
Will add comment next in PR. To avoid PR build wait. I'm planning to raise PR with this package on templating repo today.